main.go

```diff
@@ -42,7 +44,9 @@ func init() {
 
 	_ = elasticservingv1.AddToScheme(scheme)
 
-	_ = istio.AddToScheme(scheme)
```

Comment:
So we are not using istio directly any longer?

Reply:
No, I just change istio.AddToScheme to v1alpa3.AddToScheme. Since the istio reconciller function is in package called istio, naming "istio.io/client-go/pkg/apis/networking/v1alpha3" also istio may be confusing.